feat(modeles): migration en DSFR de la page qui liste les modèles de documents

m-maillot opened the pull request:
closes #6529

Viczei commented on packages/code-du-travail-frontend/src/modules/modeles-de-courriers/__tests__/LetterModels.test.tsx:
```diff
@@ -0,0 +1,93 @@
+    expect(themeOptions[0]).toHaveTextContent("Tous les thèmes");
+    expect(themeOptions[1]).toHaveTextContent("Thème A");
+    expect(themeOptions[2]).toHaveTextContent("Thème B");
+    expect(container).toMatchSnapshot();
```
Est-ce que l'on veut vraiment garder les tests snapshots ?

m-maillot replied:
Là je dois avouer que les tests me semblent assez complet. Mais comme on l'a fait pour chaque composant, j'ai gardé l'habitude.

m-maillot committed:
feat(modeles): migration en DSFR de la page qui liste les modèles de documents

# Conflicts:
#	packages/code-du-travail-frontend/__tests__/__snapshots__/modeles-de-courriers.test.tsx.snap
